scripts/consent_form_json.py

```diff
@@ -120,6 +54,5 @@ def format_consent_form(filename):
                 site_info['paragraphs'].append(row[1].strip(" "))
     return site_info
 
-
 if __name__ == '__main__':
-    main()
+     main()
```

Comment:
In general, please do not remove trailing newlines from ends of files.

This setting may be helpful for the future:
http://stackoverflow.com/questions/36043061/automatically-add-newline-on-save-in-pycharm

See context: 
http://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline